build failed
```
======================================================================
FAIL: test_trigger (custom.enikshay.integrations.ninetyninedots.tests.test_repeaters.TestUpdatePatientRepeater)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vendor/lib/python2.7/site-packages/django/test/utils.py", line 216, in inner
    return test_func(*args, **kwargs)
  File "/mnt/commcare-hq-ro/custom/enikshay/integrations/ninetyninedots/tests/test_repeaters.py", line 137, in test_trigger
    self.assertEqual(1, len(self.repeat_records().all()))
AssertionError: 1 != 0
-------------------- >> begin captured logging << --------------------
```

--- a/corehq/apps/repeaters/models.py
+++ b/corehq/apps/repeaters/models.py
@@ -361,9 +361,13 @@ class CaseRepeater(Repeater):
     friendly_name = _("Forward Cases")
 
     def allowed_to_forward(self, payload):
-        allowed_case_type = not self.white_listed_case_types or payload.type in self.white_listed_case_types
-        allowed_user = self.payload_user_id(payload) not in self.black_listed_users
-        return allowed_case_type and allowed_user
+        return self._allowed_case_type(payload) and self._allowed_user(payload)
+
+    def _allowed_case_type(self, payload):
+        return not self.white_listed_case_types or payload.type in self.white_listed_case_types
+
+    def _allowed_user(self, payload):
+        return self.payload_user_id(payload) not in self.black_listed_users
 
     def payload_user_id(self, payload):
         # get the user_id who submitted the payload, note, it's not the owner_id

--- a/custom/enikshay/integrations/ninetyninedots/repeaters.py
+++ b/custom/enikshay/integrations/ninetyninedots/repeaters.py
@@ -29,7 +29,7 @@ class NinetyNineDotsRegisterPatientRepeater(CaseRepeater):
 
     def allowed_to_forward(self, case):
         # checks whitelisted case types and users
-        allowed_case_types_and_users = super(NinetyNineDotsRegisterPatientRepeater, self).allowed_to_forward(case)
+        allowed_case_types_and_users = self._allowed_case_type(case) and self._allowed_user(case)
         case_properties = case.dynamic_case_properties()
         enabled = case_properties.get('dots_99_enabled') == 'true'
         not_registered = (
@@ -54,7 +54,7 @@ class NinetyNineDotsUpdatePatientRepeater(NinetyNineDotsRegisterPatientRepeater)
 
     def allowed_to_forward(self, case):
         # checks whitelisted case types and users
-        allowed_case_types_and_users = super(NinetyNineDotsUpdatePatientRepeater, self).allowed_to_forward(case)
+        allowed_case_types_and_users = self._allowed_case_type(case) and self._allowed_user(case)
         if not allowed_case_types_and_users:
             return False
 

